packages/wdio-sync/webdriverio.d.ts

```diff
@@ -27,9 +27,10 @@ declare namespace WebdriverIO {
     }
 
     interface BrowserObject extends WebDriver.ClientOptions, WebDriver.Client, WebdriverIO.Browser { }
+    interface MultiRemoteBrowser extends WebDriver.ClientOptions, WebDriver.Client, WebdriverIO.Browser { }
 }
 
-declare var browser: WebdriverIO.BrowserObject;
+declare var browser: WebdriverIO.BrowserObject | WebdriverIO.MultiRemoteBrowserObject;
```

Comment:
Typescript is smart enough to conditionally detect child type of union type base on property value difference.
In this case, the property is `isMultiremote`.
In `BrowserObject`, its type is `isMultiremote?: false`
In `MultiRemoteBrowserObject`, its type is `isMultiremote: true`.
So when you check conditions like below, correct type will be detected
```
if(!global.browser.isMultiremote) {
   const a = global.browser // type is detected as WebdriverIO.BrowserObject
} else {
   const b = global.browser // type is detected as WebdriverIO.MultibrowserObject
}
```